cloud

```diff
@@ -702,23 +719,28 @@ function scheduler_job_withdraw() {
 }
 
 # Deploy to scheduler
-# Args: project prefix trigger_topic
+# Args: project prefix load_queue_trigger_topic pick_notifications_trigger_topic
 function scheduler_deploy() {
     declare -r project="$1"; shift
     declare -r prefix="$1"; shift
-    declare -r trigger_topic="$1"; shift
+    declare -r load_queue_trigger_topic="$1"; shift
+    declare -r pick_notifications_trigger_topic="$1"; shift
     # Deploy the jobs
-    scheduler_job_pubsub_deploy "$project" "${prefix}trigger" \
-                                "$trigger_topic" '* * * * *' '{}'
+    scheduler_job_pubsub_deploy "$project" "${prefix}load_queue_trigger" \
+                                "$load_queue_trigger_topic" '* * * * *' '{}'
+    scheduler_job_pubsub_deploy "$project" "${prefix}pick_notifications_trigger" \
+                                "$pick_notifications_trigger_topic" \
+                                 '30 * * * *' '{}'
 }
 
 # Withdraw from the scheduler
 # Args: project prefix
 function scheduler_withdraw() {
     declare -r project="$1"; shift
     declare -r prefix="$1"; shift
-    declare -r trigger_topic="$1"; shift
-    scheduler_job_withdraw "$project" "${prefix}trigger" "$trigger_topic"
+    declare -r load_queue_trigger_topic="$1"; shift
+    scheduler_job_withdraw "$project" "${prefix}trigger"\
+                           "$load_queue_trigger_topic"
```

Comment:
Waaaait, I was wrong, we don't really need passing the third argument to `scheduler_job_withdraw` at all. It doesn't care which topic the Cloud Scheduler is sending its messages to, when removing the job, it only needs the project name and the job name to remove it. Thus we don't need the second commit :man_facepalming: Please remove it. Please also remove passing `"$triger_topic"` to `scheduler_job_withdraw` in the first commit.

Sorry, I confused you. Comes from dipping into things for half an hour only and trying to do it quick :grin:

This still has another problem, though: we need to update the jobs we're withdrawing here, to match what we're deploying in `scheduler_deploy`. That is, we need to withdraw both `"${prefix}load_queue_trigger"` and `"${prefix}pick_notifications_trigger"` here.